src/agent/agent.c

```diff
@@ -1805,47 +1805,51 @@ static bool ensure_orch_address(Agent *agent) {
         if (agent->orch_addr != NULL) {
                 return true;
         }
-
-
         if (agent->host == NULL) {
                 hirte_log_errorf("No manager host specified for agent '%s'", agent->name);
                 return false;
         }
 
+        char *ip_address = agent->host;
+        bool host_is_ipv4 = isIPv4Addr(ip_address);
+        bool host_is_ipv6 = isIPv4Addr(ip_address);
```

Comment:
isIPv6addr()?

Reply:
The dreaded copy-paste-error
Good catch, thanks!